### DISCUSSION TITLE
Logging to a file target with NLog and fo-dicom 5.0.3

### DISCUSSION THREAD

**Discussion by azzurro1971** [fo-dicom/fo-dicom#1535] [Q&A]
I have problems with logging with fo-dicom version 5.0.3 and NLog. I would like to log to a file target but it doesn't work. Is there a sample available somewhere?

Thank you very much.

**Comment by amoerie** [comment]
Hi @azzurro1971 

The latest fo-dicom simply works with Microsoft.Extensions.Logging
So if you have NLog setup with that, fo-dicom should already work.

If you don't use Microsoft.Extensions.Logging (e.g. you're still in .NET Framework), I would recommend implementing your own LogManager. You can look at our old https://github.com/fo-dicom/fo-dicom/blob/5.0.3/Logging/FO-DICOM.NLog/NLogManager.cs for an example on how it used to work.
